src/Sylius/Bundle/AdminBundle/Resources/config/app/config.yml

```diff
@@ -670,39 +670,39 @@ sylius_ui:
             blocks:
                 before_customer_information_legacy:
                     template: "@SyliusUi/Block/_legacySonataEvent.html.twig"
-                    priority: 55
```

Comment:
Legacy events had `x5` priority in purpose, to make a distinction from the normal events IIRC. What is more, such a change would rearrange the whole panel for people who already injected some UI blocks into the existing structure. I'm ok with your proposal, but I would prefer to stay with old priorities (just change which block is assigned to the given priority,

Reply:
You mean for this it should be `95` and not `90`?

Reply:
I mean it should stay 55. We should revert your adjustment to change priorities range from 90 to 10 and bring back old one (55 to 5). After that, you may apply your changes just adjusting existing priorities instead of rebuilding the whole structure.

Reply:
ok! I'll do that.